rosbag2_transport/src/rosbag2_transport/player.cpp

```diff
@@ -658,6 +690,20 @@ void Player::create_control_services()
     {
       response->success = play_next();
     });
+  srv_play_for_ = create_service<rosbag2_interfaces::srv::PlayFor>(
+    "~/play_for",
```

Comment:
I think we should remove  `srv_play_for`. Since now `srv_play` fully cover it's case.